lib/iris/tests/unit/cube/test_Cube.py

```diff
@@ -449,24 +449,49 @@ def test_different_array_attrs_incompatible(self):
 
 class Test_aggregated_by(tests.IrisTest):
     def setUp(self):
-        self.cube = Cube(np.arange(11))
+        self.cube = Cube(np.arange(44).reshape(4, 11))
+
         val_coord = AuxCoord([0, 0, 0, 1, 1, 2, 0, 0, 2, 0, 1],
                              long_name="val")
         label_coord = AuxCoord(['alpha', 'alpha', 'beta',
                                 'beta', 'alpha', 'gamma',
                                 'alpha', 'alpha', 'alpha',
                                 'gamma', 'beta'],
                                long_name='label', units='no_unit')
-        self.cube.add_aux_coord(val_coord, 0)
-        self.cube.add_aux_coord(label_coord, 0)
+        simple_agg_coord = AuxCoord([1, 1, 2, 2], long_name='simple_agg')
```

Comment:
Why is this simple?

Reply:
Just that the sequence `1, 1, 2, 2` is simple (i.e. monotonic and both numbers used twice) compared with the other coords used for aggregation (`0, 0, 0, 1, 1, 2, 0, 0, 2, 0, 1` and `'alpha', 'alpha', 'beta', 'beta', 'alpha', 'gamma', 'alpha', 'alpha', 'alpha',  'gamma', 'beta'`).  Happy to change the names if there is something  more appropriate.